src/switchImplIntf.ml

```diff
@@ -0,0 +1,60 @@
+open Import
+
+module Lsp : sig
+  val switch : LanguageClient.t -> TextDocument.t -> string option Promise.t
+end = struct
+  let switch client document =
+    let targetFileName : string option Promise.t =
+      LanguageClient.sendRequest client ~meth:"ocaml/didSwitchImplIntf"
+        ~data:(document : TextDocument.t)
+        ()
+    in
+    targetFileName
+    |> Promise.catch (fun (_ : Promise.error) -> Promise.return None)
```

Comment:
It seems wrong to swallow this error. What's wrong with just letting it bubble up?

Reply:
Making it return None allows the Fallback version to be used in `tryFallback`

Reply:
Yeah, but it seems wrong to add code to work around buggy behavior in the server.